### ISSUE TITLE
Failed build frontend

### ISSUE THREAD

**Issue by korol-sas** [Colm3na/polkastats-ng#13]
ERROR in ../node_modules/@polkadot/api/rx/Api.js 108:2
Module parse failed: Unexpected character '#' (108:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| export class ApiRx extends ApiBase {
>   #isReadyRx;
|   /**
|    * @description Create an instance of the ApiRx class
 @ ../node_modules/@polkadot/api/rx/index.js 3:0-33 3:0-33
 @ ../node_modules/@polkadot/api/bundle.js
 @ ../node_modules/@polkadot/api/index.js
 @ ../node_modules/babel-loader/lib??ref--2-0!../node_modules/@nuxt/components/dist/loader.js??ref--0-0!../node_modules/vue-loader/lib??vue-loader-options!./layouts/default.vue?vue&type=script&lang=js&
 @ ./layouts/default.vue?vue&type=script&lang=js&
 @ ./layouts/default.vue
 @ ./node_modules/.cache/nuxt/App.js
 @ ./node_modules/.cache/nuxt/index.js
 @ ./node_modules/.cache/nuxt/client.js
 @ multi ../node_modules/@nuxt/components/lib/installComponents.js ./node_modules/.cache/nuxt/client.js

**Comment by korol-sas** [comment]
# node -v
v14.19.0
# yarn -v
1.22.17

**Comment by korol-sas** [comment]
https://github.com/polkadot-js/api/issues/4500

**Comment by mariopino** [comment]
Thanks, I'm in the middle of a refactor of PolkaStats and will deal with this tomorrow!

**Comment by mariopino** [comment]
This is fixed in PR #12, relevant changes are:

- Execute `yarn workspace frontend add --dev @open-wc/webpack-import-meta-loader` 
- Edit `build` section of `nuxt.config.js` file:
```
  // Build Configuration (https://go.nuxtjs.dev/config-build)
  build: {
    transpile: [
      '@polkadot/vue-identicon',
      '@polkadot/x-ws',
      '@polkadot/api',
      '@polkadot/api-derive',
      '@polkadot/keyring',
      '@polkadot/networks',
      '@polkadot/rpc-augment',
      '@polkadot/rpc-core',
      '@polkadot/rpc-provider',
      '@polkadot/types',
      '@polkadot/types-known',
      '@polkadot/ui-shared',
      '@polkadot/util',
      '@polkadot/util-crypto',
      '@polkadot/x-bigint',
      '@polkadot/x-fetch',
      '@polkadot/x-global',
      '@polkadot/x-randomvalues',
      '@polkadot/x-textdecoder',
      '@polkadot/x-textencoder',
      '@polkadot/x-ws',
    ],
    extend(config, ctx) {
      config.module.rules.push({
        test: /\.js$/,
        loader: require.resolve('@open-wc/webpack-import-meta-loader'),
      })
      config.module.rules.push({
        test: /\.mjs$/,
        include: /node_modules/,
        type: 'javascript/auto',
      })
      // https://github.com/nuxt/nuxt.js/issues/1142
      config.resolve.alias.vue = 'vue/dist/vue.common'
    },
  },
```
